index.html

```diff
@@ -211,44 +197,44 @@ <h1>About me</h1>
     </div>
   </section>
   <!-- Libraries -->
-  <section class="lib-container">
-    <div class="lib-grid">
-      <div class="lib-card">
-        <img src="./images/icons/Icons1.png" alt="" />
-        <h1>Languages</h1>
-        <ul>
-          <li>JavaScript</li>
-          <li>Ruby</li>
-          <li>HTML</li>
-          <li>CSS</li>
-        </ul>
-      </div>
-      <div class="lib-card">
-        <img src="./images/icons/Icons2.png" alt="" />
-        <h1>Frameworks</h1>
-        <ul>
-          <li>Boostrap</li>
-          <li>Ruby on rail</li>
-          <li>RSpec</li>
-          <li>Capybara</li>
-          <li>Selenium</li>
-        </ul>
-      </div>
-      <div class="lib-card">
-        <img src="./images/icons/Icons3.png" alt="" />
-        <h1>Skills</h1>
-        <ul>
-          <li>CodeKit</li>
-          <li>GitHub</li>
-          <li>Codepen</li>
-          <li>GitLab</li>
-          <li>Terminal</li>
-        </ul>
-      </div>
-    </div>
+  <section class="library-container">
+    <ul class="card">
```

Comment:
- [ ] It looks like recent changes after after reviewer's comments have affected the design 😢 , please check: 

Before  | After
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/144696805-6e22e4df-50c6-4bda-82be-207c730cfece.png) | ![image](https://user-images.githubusercontent.com/77038610/144696809-96739e72-ea10-47eb-a0bd-f74bbfea2739.png)

More precisely, listed items are going out of container, and the font color is different:

![image](https://user-images.githubusercontent.com/77038610/144696862-a28c6d97-a053-4891-b541-c0974fd1bd43.png)

Please return previous design 🥺

Reply:
I understand your point. The text and colour didn't align with the Figma design which is quite understandable and I will work on it. For the overlapping, I am sorry, but you obviously did not view the page using your developer's tool which is a more accurate/recommended way to ascertain if the project meets the requirement or not. Besides, we are designing for desktop and not for laptop. See the page when viewed using the developer's tool which is qy=uite different from what you attached here. Thank you.

![page](https://user-images.githubusercontent.com/48631109/144698059-3594dfbc-59d3-47cb-8745-c4a818b7f97c.PNG)

Reply:
@denscholar to be honest I am a little bit offended by your comment, but that is ok. I was checking everything with developer tools, and of course I know that it is more accurate/recommended  😉 .  Here is the screen record: 

https://user-images.githubusercontent.com/77038610/144698651-e2ff2ead-1d06-4632-91a3-4cacd504ef73.mp4

As a developer you should consider different screen sizes, for example mine is more than`1440px` and I have this: 

![image](https://user-images.githubusercontent.com/77038610/144698709-4e82b6de-76ea-4e91-b097-52f6069cea8b.png)

Reply:
I am very sorry if you feel offended. I am currently working on the feedback.